### DISCUSSION TITLE
pull in sections only

### DISCUSSION THREAD

**Discussion by winne617** [DeabLabs/cannoli#5] [Ideas]
hey, would it be possible to implement the option to link to a specific heading in a note, like [[PAGE-NAME#Heading]]?

cheers

**Comment by blindmansion** [comment]
I haven't implemented that in variables yet, so you couldn't do a dynamic reference to a page like that, or write to a particular heading.

However, you can do a normal markdown embedding "![[Note#heading]]" and it will render for the LLM to see, the same way you see it in obsidian.

Hopefully that's a good enough workaround for now! That's on my list to add.
